<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>budget</class>
 <widget class="QMainWindow" name="budget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BudgetWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #979d9a;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="amountLineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #00ffff;             
	text-decoration: underline;
	font: 700 italic 9pt &quot;Arial&quot;;
    font-size: 30px;                
    font-weight: bold;        
    letter-spacing: 2px;          
    text-align: center;              
    background-color: transparent;    

}
</string>
    </property>
    <property name="text">
     <string>Set Your Budgets</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BudgetClear">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>330</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 700 italic 9pt &quot;Arial&quot;;
    background-color: #99ff99;
    border: 1px solid #003300;
    border-radius: 5px;
    padding: 5px;
    font-size: 14px;
    color: #003300;
}
QPushButton:hover {
    background-color: #66cc66;
}
</string>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>250</y>
      <width>381</width>
      <height>231</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>category</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Amount</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>date</string>
     </property>
    </column>
   </widget>
   <widget class="QComboBox" name="categoryComboBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Select a Category</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Food</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Entertainment</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Traveling</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Clothing</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Education equipment</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>University fess</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Others</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Amount</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Category</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addBudgetData">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>330</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 700 italic 9pt &quot;Arial&quot;;
    background-color: #99ff99;
    border: 1px solid #003300;
    border-radius: 5px;
    padding: 5px;
    font-size: 14px;
    color: #003300;
}
QPushButton:hover {
    background-color: #66cc66;
}
</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BudgetBack">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 700 italic 9pt &quot;Arial&quot;;
    background-color: #55aaff;
    border: 1px solid #003300;
    border-radius: 5px;
    padding: 5px;
    font-size: 14px;
    color: #000000;
}
QPushButton:hover {
    background-color: #aaffff;
}
</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>180</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 700 italic 9pt &quot;Arial&quot;;
    background-color: #aaaaff;
    border: 1px solid #003300;
    border-radius: 5px;
    padding: 5px;
    font-size: 14px;
    color: #003300;
}
QPushButton:hover {
    background-color:#e2ceff;
}
</string>
    </property>
    <property name="text">
     <string> Show Budget Details</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>510</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Total Budget : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="totalBudgetLineEdit">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>510</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>241</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>20</width>
      <height>581</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>321</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #00ffff;             
	text-decoration: underline;
	font: 700 italic 9pt &quot;Arial&quot;;
    font-size: 30px;                
    font-weight: bold;        
    letter-spacing: 2px;          
    text-align: center;              
    background-color: transparent;    

}
</string>
    </property>
    <property name="text">
     <string>Show Budget Plans</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>130</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Select a Month :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="monthComboBox">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>130</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Select a month</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>January</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>February</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>March</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>April</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>June</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>July</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>August</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>September</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>October</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>November</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>December</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
